src/unittest/python/pmf_tests.py

```diff
@@ -144,3 +181,38 @@ def test_at_zero(self):
         for sigmoid in self.sigmoids:
             with self.subTest(sigmoid=sigmoid):
                 self.assertAlmostEqual(sigmoid(0, 0, 1), 0.5)
+
```

Comment:
I don't quite understand what's going on here. I can tell it's some sort of automated test, but not why they are necessary or how they work.

Reply:
All unit tests in python have to start with `test_` (otherwise they don't get recognized). It's good practice to have the name of the test function be a "should do this and that" (easier to understand what failed). The previous test is badly named. But it's not something to fix now.